pkg/ingress/ingress.go

```diff
@@ -192,18 +192,20 @@ func (c *ingressController) handleSyncErr(obj interface{}, err error) {
 	event := ev.Object.(kube.IngressEvent)
 	namespace, name, errLocal := cache.SplitMetaNamespaceKey(event.Key)
 	if errLocal != nil {
-		log.Errorf("invalid resource key: %s", event.Key)
+		log.Errorw("invalid resource key",
```

Comment:
I think still reserve the `key` in the log will be better.

Reply:
`cache.SplitMetaNamespaceKey` return `errLocal` include `key`
``` go
return "", "", fmt.Errorf("unexpected key format: %q", key)
```